Server/database/query/User/getPersonalinfo.js

```diff
@@ -0,0 +1,10 @@
+const connection = require('../../connection');
+
+module.exports = (id) => {
+  const sql = {
+    text:
+    'SELECT users.display_name , users.phone ,users.personal_status ,users.number_of_individuals ,users.number_of_devices, address.town , address.city FROm users INNER JOIN address ON address.id = users.address_id WHERE users.id =$1;',
```

Comment:
replace  FRom with uppercase word (FROM)

Reply:
ok I will do it